eslint.config.js

```diff
@@ -0,0 +1,61 @@
+const tseslint = require('typescript-eslint');
+const eslint = require('@eslint/js');
+const prettierPluginRecommended = require('eslint-plugin-prettier/recommended');
+const typescriptEslint = require('@typescript-eslint/eslint-plugin');
+const globals = require('globals');
+
+module.exports = tseslint.config(
+  eslint.configs.recommended,
+  tseslint.configs.recommended,
```

Comment:
I initially tried `tseslint.configs.recommendedTypeChecked` here instead but there were a fare amount of issues that I wouldn't have the time nor context to adequately sort out. But you all may wanna look into enabling it at some point.

See docs [here](https://typescript-eslint.io/getting-started/typed-linting).

Reply:
Just to clarify, you got the config working, but that resulted in quite a few linter errors?

Reply:
Yep, that's right. They didn't seem that serious or difficult to solve though, a lot of them were seemingly unnecessary `await`s and implicit use of `any`s.